modules/base/rendering/screenspaceimagelocal.cpp

```diff
@@ -68,8 +70,6 @@ ScreenSpaceImageLocal::ScreenSpaceImageLocal(const ghoul::Dictionary& dictionary
 {
     const Parameters p = codegen::bake<Parameters>(dictionary);
 
-    // @TODO (abock, 2021-02-02) Should this be the name variable? The identifier wasn't
```

Comment:
This is a discussion to have @alexanderbock 

Apparently, we let the users create screen space renderables without an identifier and for most of the types we generate one using code like this one here (although, not for all...)

I honestly think that we can remove this code. Haven't found anywhere where it is being used, we seem to always add identifiers.  But it should be carefully done and investigated